lib/fog/bin/google.rb

```diff
@@ -24,25 +24,25 @@ def class_for(key)
 
     def [](service)
       @@connections ||= Hash.new do |hash, key|
-        hash[key] = case key
-                    when :compute
-                      Fog::Logger.warning("Google[:compute] is not recommended, use Compute[:google] for portability")
-                      Fog::Compute.new(:provider => "Google")
-                    when :dns
-                      Fog::Logger.warning("Google[:dns] is not recommended, use DNS[:google] for portability")
-                      Fog::DNS.new(:provider => "Google")
-                    when :monitoring
-                      Fog::Google::Monitoring.new
-                    when :sql
-                      Fog::Google::SQL.new
-                    when :pubsub
-                      Fog::Google::Pubsub.new
-                    when :storage
-                      Fog::Logger.warning("Google[:storage] is not recommended, use Storage[:google] for portability")
-                      Fog::Storage.new(:provider => "Google")
-                    else
-                      raise ArgumentError, "Unrecognized service: #{key.inspect}"
-                    end
+        case key
+        when :compute
+          Fog::Logger.warning("Google[:compute] is not recommended, use Compute[:google] for portability")
```

Comment:
Nit: Is this still an accurate error with the swap?

Reply:
No, it isn't. I dropped these warnings.

I don't even know what `fog/bin/google` is used for.